src/PKSim.Infrastructure/Serialization/WorkspaceLayoutPersistor.cs

```diff
@@ -17,29 +17,35 @@ public class WorkspaceLayoutPersistor : IWorkspaceLayoutPersistor
       private readonly IWorkspaceLayoutMetaDataToWorkspaceLayoutMapper _workspaceLayoutMapper;
 
       public WorkspaceLayoutPersistor(IWorkspaceLayoutToWorkspaceLayoutMetaDataMapper workspaceLayoutMetaDataMapper,
-                                      IWorkspaceLayoutMetaDataToWorkspaceLayoutMapper workspaceLayoutMapper)
+         IWorkspaceLayoutMetaDataToWorkspaceLayoutMapper workspaceLayoutMapper)
       {
          _workspaceLayoutMetaDataMapper = workspaceLayoutMetaDataMapper;
          _workspaceLayoutMapper = workspaceLayoutMapper;
       }
 
       public void Save(IWorkspaceLayout workspaceLayout, ISession session)
       {
-
-         //first remove old layout if existing
-         var layoutsFromDb = session.CreateCriteria<WorkspaceLayoutMetaData>().List<WorkspaceLayoutMetaData>();
-         layoutsFromDb.Each(session.Delete);
+         var layoutFromDb = loadLayoutFromDb(session);
```

Comment:
I realized that the content was always deleted and added. A little stupid and also annoying when looking at what is changing when saving a project. That way, we update the existing layout if there is already one instead of saving it again